teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @specialization = specialization
```

Comment:
### [ Optional ]
Hi @BranBayou, what if you could have left the super to be like this: `super(age, parent_permission, name)` without reassigning values to the properties?

Reply:
Hi @AmaduKamara,
Thank you for your constructive feedback. I will do that.